docs/quick-tutorials/k8s-kafka-mtls.mdx

```diff
@@ -111,23 +122,19 @@ kubectl apply -f https://docs.otterize.com/code-examples/kafka-mtls/kafkaserverc
 </Tabs>
 </details>
 
-Upon applying the KafkaServerConfig, an ACL will configure Kafka to allow only authenticated access to the *transactions* topic by denying all anonymous access.
-This will be the base state, from which we will gradually roll out secure access to Kafka.
-
-```bash
-kubectl logs -n kafka statefulset/kafka | grep "Processing Acl change" | grep ANONYMOUS | tail -n 1
-```
-You should see the following output:
-```
-[2023-05-18 11:49:14,230] INFO Processing Acl change notification for ResourcePattern(resourceType=TOPIC, name=transactions, patternType=LITERAL), versionedAcls : Set(User:ANONYMOUS has DENY permission for operations: ALL from hosts: *, User:* has ALLOW permission for operations: ALL from hosts: *), zkVersion : 0 (kafka.security.authorizer.AclAuthorizer)
-```
+We've chosen to allow unauthenticated access to both topics to begin with. Initially, we're not requiring declared intents so everything is open here. This will be the base state, from which we can gradually secure access to specific topics.
 
 ## Deploy clients
 
-Clients will authenticate to Kafka using mTLS. Otterize makes this easy, requiring just 3 simple changes to the client pod spec:
+Our simple example consists of two client pods:
+- One named "**client**".
+- And one named "**client-other**".
+
+These clients will eventually authenticate to Kafka using mTLS, but we haven't turned that on just yet. Otterize makes this easy, requiring just 3 simple changes to the client pod spec:
```

Comment:
The clients are authenticating to Kafka with mTLS already at this point as we aren't using a plaintext connection here. It's just that this isn't required by Kafka. There's no need to switch from plaintext to SSL just to show this, but just update how we phrase this.